src/modules/CRUD/components/Details/Details.js

```diff
@@ -203,7 +206,7 @@ class Details extends React.Component {
           {!Object.keys(feature).length && paramId !== ACTION_CREATE ? (
             <Loading spinner />
           ) : (
-            <>{this.renderContent()}</>
+            this.renderContent()
```

Comment:
```js
{!Object.keys(feature).length && paramId !== ACTION_CREATE
  ? <Loading spinner />
  : this.renderContent()}
```

```js
{(
  !Object.keys(feature).length && paramId !== ACTION_CREATE
    ? <Loading spinner />
    : this.renderContent()
)}
```

Also, `!Object.keys(feature).length && paramId !== ACTION_CREATE` could be turn into _(pre-computed)_ more meaningful const(s). Couldn't they?

```js
{(
  myPrettyConstName
    ? <Loading spinner />
    : this.renderContent()
)}
```